helpers/errorHandlers.js

```diff
@@ -5,25 +5,11 @@ module.exports = {
     let status, message;
     // check for daemon being turned off
     if (error.code === 'ECONNREFUSED') {
-      status = 503;
+      status = 200;
```

Comment:
I am trying to create a pattern where server responded with a 200 as long as it processed the request correctly, and then handled the success of the data portion of the request with a `success` and a `message` or `data` field.  I.e. `{"success": "false", "message": "connection refused.  The daemon may not be running"}`.  This way, if the client wanted some data but the daemon was off, it would get a `200` response that the request was completed, and then the client could check the response to see if it was successful or not and whether the data is present. My goal was to help myself (and api users) to distinguish between an error where something was unavailable but the request was handled properly (resulting in a `200`) vs a request that I thought would be handled but wasn't, resulting in some other error.  Thoughts?

Reply:
I think proper error codes would still be preferable. Even if the request is handled properly, there is still an error. If I see a 200 response, I assume my request was successful. The `fetch` api handles errors really nicely too, so I think if anything you are creating more work.

```
fetch('spee.ch/xxx')
  .then((data) => ...)         // successful
  .catch((error) => ...)      // not successful

fetch('spee.ch/xxx')
  .then((data) => {           // maybe successful?
    if (!data.success) {
      throw new Error()
   }
    ...                                 // ok now I know it was successful
  })
  .catch((err) => ... )       // not successful
```
In my opinion it doesn't really matter if a request was handled successfully. If the data doesn't make it, it still fails. You can still add info with the error message and code.